AEPMessaging/Sources/Messaging.swift

```diff
@@ -191,17 +198,55 @@ public class Messaging: NSObject, Extension {
     /// Called on every event, used to allow processing of the Messaging rules engine
     private func handleWildcardEvent(_ event: Event) {
         rulesEngine.process(event: event)
+
+        let qualifiedContentCardsBySurface = getPropositionsFromContentCardRulesEngine(event)
+        for (surface, propositions) in qualifiedContentCardsBySurface {
+            addOrReplaceContentCards(propositions, forSurface: surface)
+        }
+    }
+
+    /// Prevents multiple propositions from being in `contentCardsBySurface` at the same time
+    /// If an existing entry for a proposition is found, it is replaced with the value in `cbePropositions`.
```

Comment:
typo: it is replaced with the value in `_propositions`

Reply:
actually just `propositions` i think - it the name of the param, and the param wins here